apps/andi/lib/andi/event/event_handler.ex

```diff
@@ -107,9 +115,11 @@ defmodule Andi.Event.EventHandler do
 
   def handle_event(%Brook.Event{
         type: user_organization_associate(),
-        data: %UserOrganizationAssociate{subject_id: subject_id, org_id: org_id, email: _email} = data,
+        data: %UserOrganizationAssociate{subject_id: subject_id, org_id: org_id} = data,
```

Comment:
Why remove email? Are we sure this isn't an intended hydra filter?

Reply:
whoops, just saw the underscore and autopiloted